src/main/resources/sql/select2-1-select2-5.sql

```diff
@@ -0,0 +1,35 @@
+-- Find the names of buyers, whose age is from 16 to 35
+SELECT c.name
+FROM customers c
+WHERE c.age BETWEEN 16 AND 35;
+
+-- Find the names of buyers whose surname ends in 'OV'
+SELECT c.name
```

Comment:
May be we should also select c.surname in order to check results?

Reply:
The task is asked to find the names of customers